site/src/App/Navigation/Navigation.css.ts

```diff
@@ -56,10 +56,46 @@ const subNavOffsetAboveMobile = style(
   }),
 );
 
+const backgroundVar = createVar();
+const shadowVar = createVar();
+const transparent = 'rgba(0, 0, 0, 0)';
+const scrollShadows = style([
+  {
+    backgroundPosition: 'center top',
+    backgroundRepeat: 'no-repeat',
+    backgroundAttachment: 'local, scroll',
+  },
+  colorModeStyle({
+    lightMode: {
+      vars: {
+        [backgroundVar]: vars.backgroundColor.body,
+        [shadowVar]: rgba(palette.grey['800'], 0.2),
+      },
+      backgroundImage: [
+        `linear-gradient(${backgroundVar} 30%, ${transparent})`,
+        `radial-gradient(farthest-side at 50% 0, ${shadowVar}, ${transparent})`,
+      ].join(', '),
+      backgroundSize: '100% 40px, 100% 18px',
+    },
+    darkMode: {
+      vars: {
+        [backgroundVar]: vars.backgroundColor.bodyDark,
+        [shadowVar]: palette.grey['800'],
+      },
+      backgroundImage: [
+        `linear-gradient(45deg, ${backgroundVar}, ${backgroundVar})`,
+        `linear-gradient(90deg, ${transparent}, ${shadowVar} 15%, ${shadowVar} 85%, ${transparent})`,
+      ].join(', '),
+      backgroundSize: '100% 2px',
+    },
```

Comment:
Just checking if this intentionally looks different from the light mode version.

<img width="331" alt="dark" src="https://github.com/seek-oss/braid-design-system/assets/3297808/223d01ef-b58b-47c5-8827-4a19418cb98d">

vs

<img width="331" alt="light" src="https://github.com/seek-oss/braid-design-system/assets/3297808/bda40c65-8fe7-40f0-8496-a09118b26089">

Reply:
Yep. Shadows arent really a recommended way of differentiating layers in dark mode. So I opted for a different treatment.

Reply:
Personally, I'd like to have it a bit bigger, especially for larger screens where it can be hard to notice.